.vib/consul/cypress/cypress.json

```diff
@@ -0,0 +1,10 @@
+{
+  "baseUrl": "http://localhost",
+  "pageLoadTimeout": 240000,
+  "defaultCommandTimeout": 60000,
+  "env": {
+    "datacenterName": "datacenter1",
+    "replicaCount": "3",
+    "containerPorts": "8400"
```

Comment:
This is a specific port (RPC), hence I would rename the env to something like `containerRpcPort`

Reply:
Changed!